File: Atcoder_#349/c/a_zero_sum_game.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);

    int a[n-1];
    for (int i = 0; i < n-1; i++) {
        scanf("%d", &a[i]);
    }

    int a_sum = 0;
    for (int i = 0; i < n-1; i++) {
        a_sum += a[i];
    }

    int result = -a_sum;
    if (result == 0) {
        result = 0;
    }

    printf("%d\n", result);

    return 0;
}

File: Atcoder_#349/javascript/a_zero_sum_game.js
function main(input) {
  const lines = input.trim().split('\n');
  const n = Number(lines[0]);
  const a = lines[1].split(' ').map(Number);

  const a_sum = a.reduce((acc, num) => acc + num, 0);

  const result = -a_sum === 0 ? 0 : -a_sum;

  console.log(result);
}
main(require("fs").readFileSync("/dev/stdin", "utf8"));
